File: lang_specs/cpp/options/class/max.cpp
class {classname}{extends_clause}
{
private:
    std::string message_;

public:
    {classname}() : message_("Hello, World!") { }
    explicit {classname}(const std::string& message) : message_(message) { }
    virtual ~{classname}() = default;
    
    const std::string& getMessage() const { return message_; }
    void setMessage(const std::string& message) { message_ = message; }
    
    virtual void {methodname}() {
        std::cout << message_ << std::endl;
    }
};

File: lang_specs/java/options/class/max.java
public class {classname} {
    private String message;
    
    public {classname}() {
        this("Hello, World!");
    }
    
    public {classname}(String message) {
        this.message = message;
    }
    
    public String getMessage() {
        return this.message;
    }
    
    public void setMessage(String message) {
        this.message = message;
    }
    
    public void greet() {
        System.out.println(this.message);
    }
    
    public static void main(String[] args) {
        {classname} instance = new {classname}();
        instance.greet();
    }
}